engine/src/test/java/org/camunda/bpm/engine/test/api/identity/IdentityServiceTest.java

```diff
@@ -1066,6 +1067,44 @@ public void shouldCreateUserWithEmptyUserId() {
     assertThat(user).isNotNull();
   }
 
+  @Test
+  public void shouldIncludePlaintextPasswordInUserToString() {
+    // given
+    User user = identityService.newUser("id");
+
+    String password = "this is a password";
+    user.setPassword(password);
+
+    // when
+    String toString = user.toString();
+
+    // then
+    assertThat(toString).doesNotContain(password);
+  }
+
+  @Test
+  public void shouldIncludeHashedPasswordAndSaltInUserToString() {
```

Comment:
Just wondering, should this be named `shouldNot...`? 🤔 
```suggestion
  public void shouldNotIncludeHashedPasswordAndSaltInUserToString() {
```